service/controller/internal/cloudconfig/tccpn.go

```diff
@@ -0,0 +1,98 @@
+package cloudconfig
+
+import (
+	"context"
+
+	infrastructurev1alpha2 "github.com/giantswarm/apiextensions/pkg/apis/infrastructure/v1alpha2"
+	"github.com/giantswarm/aws-operator/service/controller/controllercontext"
+	"github.com/giantswarm/aws-operator/service/controller/key"
```

Comment:
The `architect-orb` used to build projects in CI got just extended with a check for import blocks. See https://github.com/giantswarm/fmt/blob/master/go/imports.md.